### PR TITLE
CVR Events

### PR DESCRIPTION
**Pull request by jaceks2106**

### CODE REVIEW

File: sample/Kmd.Logic.Cvr.Client.Sample/Program.cs [hunk 1]
```diff
@@ -114,6 +114,12 @@ private static async Task Run(AppConfiguration configuration)
                 var productionUnitDetail = await cvrClient.GetProductionUnitDetailAsync(productionUnits[0].PNumber).ConfigureAwait(false);
 
                 Log.Information("Production Unit Detail data: {@ProductionUnitDetail}", productionUnitDetail);
+
+                Log.Information("Fetching company events using configuration {Name}", configuration.CvrNumber, cvrProvider.Name);
```
Comment (nakulnseerpu) [inline sample/Kmd.Logic.Cvr.Client.Sample/Program.cs:118]: Where are we using CVR number in the string?

Comment (jaceks2106) [reply sample/Kmd.Logic.Cvr.Client.Sample/Program.cs]: fixed

File: sample/Kmd.Logic.Cvr.Client.Sample/Program.cs [hunk 2]
```diff
@@ -114,6 +114,12 @@ private static async Task Run(AppConfiguration configuration)
                 var productionUnitDetail = await cvrClient.GetProductionUnitDetailAsync(productionUnits[0].PNumber).ConfigureAwait(false);
 
                 Log.Information("Production Unit Detail data: {@ProductionUnitDetail}", productionUnitDetail);
+
```
Comment (Araballi) [inline sample/Kmd.Logic.Cvr.Client.Sample/Program.cs:117]: Can we have sample for Subscribe , un subscribe and get subscribed events

**Review by Araballi — COMMENTED** [review]
Changes.md required changes

File: sample/Kmd.Logic.Cvr.Client.Sample/Program.cs (hunk 2 above)
Comment (jaceks2106) [reply sample/Kmd.Logic.Cvr.Client.Sample/Program.cs:117]: done

**Review by Araballi — APPROVED** [review]
LGTM

**Pull request merged**